src/test/java/com/kodilla/ecommercee/domain/CartItemTest.java

```diff
@@ -0,0 +1,28 @@
+package com.kodilla.ecommercee.domain;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.BDDMockito;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.math.BigDecimal;
+
+import static org.junit.Assert.*;
+
+public class CartItemTest {
+
+    @Autowired
+    Cart cart;
+    int quantity = 3;
```

Comment:
ja bym to do given wrzucił

Reply:
jak wyżej... niechlujny test na szybko robiony